src/task.c

```diff
@@ -448,6 +462,7 @@ void JL_NORETURN throw_internal(jl_value_t *exception JL_MAYBE_UNROOTED)
         }
         assert(cur_block == eh->timing_stack);
 #endif
+        // todo sanitizer
```

Comment:
@vtjnash in which stack does the exception handler run?

Reply:
this is never a task switch, it's just a longjmp

Reply:
We need to annotate that to which stack they are jumping otherwise ASAN doesn't like us.